src/models/nebula36-1/assembly.md

```diff
@@ -0,0 +1,399 @@
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to install 2.5" drives:
+
+1. [Remove the top case](#removing-the-top-case).
+2. Unscrew and remove the 2.5" drive cage's cover.
+
+![2.5" SATA drive cage cover](./img/25-drive-cover.webp)
+
+3. Remove the plastic ring from the top crossbar and slide out four screws for each drive being installed.
+
+![2.5" SATA drive screws on top crossbar](./img/25-drive-screws.webp)
+
+4. Insert two screws into each side of the 2.5" drive, then slide the drive into the drive cage, aligning the rubber mounts on the screws around the metal of the drive cage.
+    - The larger portion of the SATA connector (the power section) should be at the top, while the smaller portion (the data section) should be at the bottom.
+5. If the [optional SATA backplane](#installing-the-sata-backplane) is not present, connect the SATA data and power cables from the back of the drive cage.
+    - A cutout in the chassis allows easy access to the back of the drive cage.
+
+## Replacing the bottom case fan:
+
+The 140mm bottom case fan (`BQ SIW4-14025-LF-PWM`) is preinstalled, but can be replaced if it becomes defective. The fan can also be removed to clean its dust filter.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to replace the bottom case fan:
+
+1. [Remove the top case](#removing-the-top-case).
+    - The [side GPU brace](#removing-the-gpu-brace) can optionally be removed to make working with the fan easier.
+2. If the system is already assembled, unplug the fan from the fan splitter board or motherboard.
+3. Set the machine down on its side and unscrew the four fan screws.
+
+![Bottom fan screws](./img/bottom-fan-screws.webp)
+
+4. When reinstalling the fan, the components should be in the following order:
+    - Chassis
+    - Dust filter
+    - Acrylic spacer
+    - Fan
+
+## Installing side intake fans:
+
+nebula36 supports up to two 120mm side intake fans mounted to the GPU brace. One 120mm fan (`BQ SIW4-12025-MF-PWM`) is sold as an optional add-on at the time of purchase.
+
+### Steps to install the side intake fans:
+
+1. [Remove the top case](#removing-the-top-case).
+2. Unpack the fan and rubber mounting pins from the fan box.
+    - The plastic rings included with the rubber mounting pins are not needed.
+
+![Be Quiet! Silent Wings 4 120mm Box](./img/side-fan-box.webp)
+
+3. Place the fan in the center of the GPU brace.
+    - The spinning side of the fan should face outward, while the stationary label should face inward.
+    - The preinstalled pinholes on the fan will line up with the long holes in the GPU brace.
+4. From the back (inside of the machine), insert the rubber mounting pins through the pinholes at each corner of the fan.
+    - The GPU brace can optionally be [removed](#removing-the-gpu-brace) to make working with the rubber mounting pins easier.
+
+![Mounted side fan](./img/side-fan-mounted.webp)
```

Comment:
The way this fan is mounted, it will be an exhaust fan instead of an intake fan. Since we refer to it as an intake fan, the picture should be retaken with the fan flipped around. The description above of the stationary label facing inwards is correct though.

Reply:
Will need to get the nebula36 back and retake this photo.

Reply:
Retook the photo & dropped the replacement in.